packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -452,7 +452,15 @@ export const prepareVaultManagerKit = (
               ephemera.storedCollateralQuote = value;
             },
             onRejected() {
-              facets.helper.observeQuoteNotifier();
+              // XXX drastic action, if the quoteNotifier fails, we don't know
```

Comment:
great comment overall.

`XXX` marks tech debt. Do you have ideas for improving this? If not, it seems more like a `NOTE` or `NB` (https://en.wikipedia.org/wiki/Nota_bene)


also, small typo: ingorance.

Reply:
fixed.